### PR TITLE
Updates required for Cosmos SDK 0.45.3 and ADR-038

### PR DESCRIPTION
**Pull request by derekadams**
## Description

The following updates are required to upgrade Provenance to use Cosmos SDK 0.45.3 along with ADR-038

closes: #781

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by channa-figure — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,22 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+This release addresses issues related to IAVL concurrency and Tendermint performance that resulted in occasional panics when under high-load conditions such as replay from quicksync. In particular, nodes which experienced issues with "Value missing for hash" and similar panic conditions should work properly with this release. The underlying Cosmos SDK `0.45.3` release that has been incorporated includes a number of improvements around IAVL locking and performance characteristics.
+
+** NOTE: Although Provenance supports multiple database backends, some issues have been reported when using the `goleveldb` backend. If experiencing issues, using the `cleveldb` backend is preferred **
+
+### Improvements
+
+* Update Provenance to use Cosmos SDK 0.45.3 Release [\#781](https://github.com/provenance-io/provenance/issues/781)
+* Plugin architecture for ADR-038 + FileStreamingService plugin [\#10639](https://github.com/cosmos/cosmos-sdk/pull/10639)
+* Sporadic error: "panic: Value missing for hash" [\#611](https://github.com/provenance-io/provenance/issues/611)
```
Comment (iramiller) [inline CHANGELOG.md:54]: Might clarify that this resolves this error. First time I read this I saw it as 'there is this sporadic error'

**Review by iramiller — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (derekadams) [reply CHANGELOG.md]: Agreed.. I updated the description to clarify.

**Review by iramiller — APPROVED** [review]
LGTM. You love to see that missing hash fix finally.

**Review by channa-figure — APPROVED** [review]

**Pull request merged**